scene/2d/collision_polygon_2d.cpp

```diff
@@ -254,6 +255,9 @@ TypedArray<String> CollisionPolygon2D::get_configuration_warnings() const {
 			warnings.push_back(RTR("Invalid polygon. At least 2 points are needed in 'Segments' build mode."));
 		}
 	}
+	if (one_way_collision && Object::cast_to<Area2D>(get_parent())) {
+		warnings.push_back(RTR("The one_way_collision property will be ignored when parent is an Area2D."));
```

Comment:
Maybe phrasing it like this:
```suggestion
		warnings.push_back(RTR("The One Way Collision property will be ignored when the parent is an Area2D."));
```